src/olmo_core/model_ladder2/transformer_model_configurator.py

```diff
@@ -0,0 +1,197 @@
+import math
+import re
+from dataclasses import dataclass
+
+from olmo_core.config import DType, StrEnum
+from olmo_core.data import TokenizerConfig
+from olmo_core.distributed.parallel import DataParallelType
+from olmo_core.exceptions import OLMoConfigurationError
+from olmo_core.nn.attention import AttentionBackendName
+from olmo_core.nn.transformer import TransformerConfig
+from olmo_core.optim import OptimConfig, Scheduler
+from olmo_core.train.train_module import (
+    TransformerDataParallelConfig,
+    TransformerDataParallelWrappingStrategy,
+    TransformerTrainModule,
+    TransformerTrainModuleConfig,
+)
+from olmo_core.utils import warn_once
+
+from .base import DeviceMeshSpec, ModelConfigurator
+from .utils import format_count
+
+
+class TransformerSize(StrEnum):
+    size_190M = "190M"
```

Comment:
Just noting to myself that it looks like the smallest size takes about 30m to hit 1xC